From d7214e8a6d163d9a3f94d3b840bc0e48bdf4d0c5 Mon Sep 17 00:00:00 2001
From: Stefan Haller <stefan@haller-berlin.de>
Date: Tue, 25 Jan 2022 19:05:21 +0100
Subject: [PATCH 20/20] Skip validation of asciidoc files

This seems to sometimes fail, and since we don't care about fixing broken
documents, ignore it.
---
 Documentation/Makefile   | 2 ++
 contrib/subtree/Makefile | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/Documentation/Makefile b/Documentation/Makefile
index b629176d7d..ea5960e80c 100644
--- a/Documentation/Makefile
+++ b/Documentation/Makefile
@@ -193,6 +193,8 @@ MAN_BASE_URL = file://$(htmldir)/
 endif
 XMLTO_EXTRA += --stringparam man.base.url.for.relative.links='$(MAN_BASE_URL)'
 
+XMLTO_EXTRA += -x manpage.xsl
+
 ifdef USE_ASCIIDOCTOR
 ASCIIDOC = asciidoctor
 ASCIIDOC_CONF =
diff --git a/contrib/subtree/Makefile b/contrib/subtree/Makefile
index 6fa7496bfd..d6a32b672a 100644
--- a/contrib/subtree/Makefile
+++ b/contrib/subtree/Makefile
@@ -25,7 +25,7 @@ ASCIIDOC_HTML    = xhtml11
 ASCIIDOC_DOCBOOK = docbook
 ASCIIDOC_EXTRA   =
 XMLTO            = xmlto
-XMLTO_EXTRA      =
+XMLTO_EXTRA      = --skip-validation
 
 ifdef USE_ASCIIDOCTOR
 ASCIIDOC         = asciidoctor
-- 
2.40.1

